src/client/components/ChangesetGroupedList/ChangesetGroupedList.js

```diff
@@ -107,7 +107,7 @@ class ChangesetGroupedList extends Component {
                             padding: '10px',
                             float: 'right',
                           }}
-                        >{chunk.date}</span>
+                        >{chunk.commits[0].date}</span>
```

Comment:
Given that the array is sorted from latest to oldest (I'm not sure I can assume it is always true), this will show the date of the version as the date of the latest commit. Is this how we would like to have it?